packages/kernel/src/worker.ts

```diff
@@ -281,7 +280,6 @@ async function loadPyodideAndPackages() {
   // The following Python code is based on streamlit.web.cli.main_run().
   self.__streamlitFlagOptions__ = {
     ...streamlitConfig,
-    "browser.gatherUsageStats": false,
```

Comment:
Do you think this feature should be enabled?

Reply:
Good question. From our data side, we are able to handle and distinguish telemetry coming from stlite apps. And in the long-term if stlite gets more integrated as an official Streamlit aspect, we probably want to enable it for all stlite apps. But for this update, I think its fine to leave it disabled as default but in a way that it can be activated if it is explicitly set to `true`.

Reply:
See the update code

Reply:
Thanks!